rest_framework_nested/serializers.py

```diff
@@ -52,3 +55,37 @@ class Meta:
         field_kwargs = get_nested_relation_kwargs(relation_info)
 
         return field_class, field_kwargs
+
+
+class NestedHyperlinkedIdentityField(HyperlinkedIdentityField):
```

Comment:
The serializer wasn't meant to be in relations anyway. I've removed the code in relations.py, and left it in serializers.py.
The last fixme comment is still here, as I'm not sure which errors could happen, but I will see if I can create some tests for that.